### PR TITLE
Chat link profile

### PR DESCRIPTION
**Pull request by kulla**
See #935 Changes:

* Add link to direct message a person via chat
* Add code for rendering the motivation text (only for `/user/26217/Kulla`) – See https://frontend-git-chat-link-profile-serlo.vercel.app/user/26217/Kulla

@elbotho Ready for feedback regarding design :smile: 

Mobile: 
2021-05-25-203603_347x709_scrot

Desktop: 
2021-05-25-203524_868x551_scrot

### CODE REVIEW

**Comment by elbotho** [comment]
**style suggestions:**

message-button:
`align-self: self-start;
margin-top: 5px;
`

h1 username:
`font-weight: bold;`

motivation
`font-size: 1.3em; text-align: left;`

ProfileHeader
`margin-top: 60px; 
margin-bottom: 50px;`

File: src/data/en/index.ts [hunk 1]
```diff
@@ -170,7 +170,7 @@ export const instanceData = {
       activeSince: 'With Serlo since',
       recentActivities: 'Recent activities',
       showAllActivities: 'Show all activities',
-      lastLogin: 'Last login',
+      lastLogin: 'Las:wqt login',
```
Comment (elbotho) [inline src/data/en/index.ts]: :)

**Comment by elbotho** [comment]
Maybe we should hide the direct message button on the users own profile.

**Comment by kulla** [comment]
> Maybe we should hide the direct message button on the users own profile.

Actually I would show the the direct message button for two reasons:

1. The own profile looks like the a thrird person would see it.
2. Actually you can chat with yourself in Rocket.Chat (like in Slack or in WhatsApp)
3. People have access to a link to their chat to send it to somebody else
4. I do not want to design the profile without the chat button :see_no_evil:

**Comment by kulla** [comment]
Here are the possibilities I see the profile can be rendered without the chat button: 
2021-05-27-121142_828x439_scrot and 
2021-05-27-121201_871x368_scrot

**Comment by kulla** [comment]
@elbotho Ready for merge. Thanks for the feedback!

**Pull request merged**